### DISCUSSION TITLE
ImportError: no module named 'mini_micropyGPS'

### DISCUSSION THREAD

**Discussion by triatebr** [BIPES/BIPES#186] [General]
When using the GPS module, after configuration ... at the time of PLAY the following error message appeared
ImportError: no module named 'mini_micropyGPS'

What to do?
Has anyone ever experienced this?
block image: [link](https://drive.google.com/file/d/1zcT6XTJTu4nnxWipA0NIe7ZnBBstNqbc/view?usp=sharing)

thanks

**Comment by jetannenbaum** [comment]
This block is not properly defined and will need to be rewritten for it to work properly.  The issue you are seeing is related to not having the ability to load the library.  I'll spend some time on it in the next couple of days.

**Reply by jetannenbaum** [reply]
Which board are you using?  I found and fixed the issue for the RP2040 boards in the pico code base available online here: https://bipes.net.br/pico/ui/  I can't test the code, since I don't have the GPS module, but you can load the library and I believe you can get the position data.
